cloud/pkg/dynamiccontroller/application/application.go

```diff
@@ -326,7 +340,16 @@ func (a *Agent) doApply(app *Application) {
 }
 
 func (a *Agent) GC() {
-
+	time.AfterFunc(time.Minute, func() {
+		a.Applications.Range(func(key, value interface{}) bool {
+			app := value.(*Application)
+			lastCloseTime := app.LastCloseTime()
+			if !lastCloseTime.IsZero() && time.Since(lastCloseTime) >= time.Minute*5 {
+				a.Applications.Delete(key)
+			}
+			return true
+		})
+	})
```

Comment:
`AfterFunc`  will only be executed once after one minute, not once every minute.
Doesn't seem to fit the semantics of GC.

Reply:
fixed